khal/controllers.py

```diff
@@ -27,11 +27,12 @@
 from collections import defaultdict
 from shutil import get_terminal_size
 
-import datetime
+import datetime as dt
```

Comment:
renaming all variables called `datetime` to something else is probably a good idea, but I still don't like to `import datetime`, I prefer to either `from datetime import datetime` or `import datetime as dt` so that all `datetime`s are always `datetime.datetime`.